src/__mocks__/sdk.js

```diff
@@ -14,6 +14,10 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
+export const mockSDKMeeting = {
+  id: 'id',
```

Comment:
```suggestion
  ID: 'id',
```
However, the shape of the meeting should match that of what is [defined in the interface](https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L8). If the property has no value, then it should be set to `null`